components/utils.py

```diff
@@ -1226,9 +1226,3 @@ def verify_service_http(service_name, url, predicate=None):
         return check_http_response(url, predicate)
     except (IOError, ValueError) as e:
         ctx.abort_operation('{0} error: {1}: {2}'.format(service_name, url, e))
-
-
-def verify_port_open(service_name, port, host='localhost'):
```

Comment:
this was only used in the rabbitmq script, might as well get rid of this now that we  don't use it.